integr8ly/integreatly-operator
# [WIP] MGDAPI-4390 Investigate moving rhsso to the productised Go operator

# Issue link
https://issues.redhat.com/browse/MGDAPI-4390
## This is Spike/Investigation PR, Not for merge.
PR for merge / for new rhoam release 1.26 - please see https://github.com/integr8ly/integreatly-operator/pull/2850

# What
The community operator is deprecated and soon to become unsupported.
We need to move to the productised version of the operator.


# Verification steps
- Verify that all e2e tests pass with the product operator
- Verify that the fake version transition upgrade is feasible
- General upgrade testing is critical. Verify all e2e tests pass after upgrade.
